src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -76,7 +76,7 @@ def handle_exception(ex):  # pylint: disable=too-many-locals, too-many-statement
         if isinstance(ex, azclierror.AzCLIError):
             az_error = ex
 
-        elif isinstance(ex, JMESPathTypeError):
+        elif isinstance(ex, JMESPathError):
```

Comment:
Are all the JMESPathErrors caused by invalid value?

Reply:
I thinks so. `JMESPathError` is inherited from `ValueError`.

Reply:
just double confirm, `JMSPathTypeError` is inherited from `JMSPathError`

Reply:
Yes.